cmd/gitsighup.go

```diff
@@ -29,49 +28,55 @@ func main() {
 	signal.Notify(c, syscall.SIGHUP)
 	go config2.Refresh(c)
 
-	r := gin.Default()
-	r.PUT("/api/v1/services/:name", func(c *gin.Context) {
-		var serviceName = c.Param("name")
-		var tag = c.Query("tag")
-
-		path, ok := getPath(c, config2.GlobalConfig, serviceName)
-		if !ok {
-			return
-		}
-
-		err = action.Action(serviceName, path, tag)
-		if err == nil {
-			c.Status(http.StatusOK)
-			return
+	var namespace string
+	var service string
+	var filename string
+	var dst string
+	for _, i := range config2.GlobalConfig.Services {
+		for _, j := range i.ConfigPath {
+			filename = j.Src
+			dst = j.Dst
 		}
-
-		err = action.Restart(serviceName, path)
-
-		c.JSON(http.StatusBadRequest, map[string]string{
-			"code":    "3005",
-			"message": fmt.Sprintf("failed to update configuration: %v", err),
-		})
-	})
-
-	r.Run() // listen and serve on 0.0.0.0:8080 (for windows "localhost:8080")
+		namespace = i.NameSpace
+		service = i.Name
+	}
+	url := "http://127.0.0.1:3000/api/v1/configsrv/"
+	SendRequest(url, namespace, service, filename, dst)
```

Comment:
连接不到
connect failed:Get "http://192.168.0.143:3000/api/v1/configsrv/dev/alertmanager/alertmanager.yml": dial tcp 192.168.0.143:3000: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.PASS